src/aks-preview/azcli_aks_live_test/configs/ext_matrix_default.json

```diff
@@ -30,7 +30,8 @@
             "test_aks_create_with_monitoring_aad_auth_msi",
             "test_aks_create_with_monitoring_aad_auth_uai",
             "test_aks_enable_monitoring_with_aad_auth_msi",
-            "test_aks_enable_monitoring_with_aad_auth_uai"
+            "test_aks_enable_monitoring_with_aad_auth_uai",
+            "test_aks_create_and_update_with_managed_nat_gateway_outbound"
```

Comment:
```suggestion
            "test_aks_create_and_update_with_managed_nat_gateway_outbound",
            "test_aks_create_with_http_proxy_config"
```

Reply:
> > @kaiqzhan @FumingZhang Could you please take a look at this live test issue?
> 
> ```
> AzureKubernetesServiceScenarioTest.test_aks_create_with_http_proxy_config
> '(PreviewFeatureNotRegistered) Preview feature Microsoft.ContainerService/HTTPProxyConfigPreview not registered.'
> ```
> 
> Seems not related to my change. @FumingZhang Can I just exclude this test?

Sure, you can exclude this test case like this. This case could execute normally until 08-30, seems that the validation in RP side takes effect from 08-31

Reply:
Thanks for checking. Updated.